### DISCUSSION TITLE
Hard fork required, and rebranding

### DISCUSSION THREAD

**Discussion by didlie** [dogecoin/dogecoin#2739] [General]
Problems:

1) The 10,000 reward is exploited by DMr3... miner through an augmented older core version, and many empty blocks are mined and excessive dilution reducing the value of Dogecoin.

2) The Dog brand is depricating, and unnecessary as "Doge" also refers to "Duke."

Dogecoin should be forked to enforce a strict limit on the number of blocks per minute, increase the maximum block size and transactions per block, reduce the dilution to ensure appreciation, and rebrand to remove the "Shibu Inu" depricating dog brand with some icon related to the crown of a duke.

Evolution of the network is important and necessary for the long-term sustainability of the network, appreciation of the Doge unit and respectability and future application of the network as a world-wide currency and/or method of exchange.

These upgrades can only evolve through a hard fork, as the exploitation of the network by this repository's controllers is too profitable for them to accept change that benefits the community of holders in general.

**Comment by patricklodder** [comment]
It is odd that you are posting this here. The permissionless nature of Dogecoin allows you to do this without this post.

Got code?

**Reply by didlie** [reply]
https://github.com/didlie/doge

**Reply by patricklodder** [reply]
No code has been changed there yet, per [the diff](https://github.com/dogecoin/dogecoin/compare/master...didlie:master) Updating the readme does not constitute a hard fork.

A few notes:
- Please note that several trademark owners are going to hurt your rebranding proposal.
- It may be prudent to propose a solution for the issue you are perceiving before going into rebranding. Look at item 29 from [the questionnaire](https://assets.website-files.com/5d766f847039d787f8a99a02/609998f89636a3f99c8429c6_CRC%20Securities%20Law%20Framework.pdf) that the [CRC](https://www.cryptoratingcouncil.com) uses to have a clear reason, identified by the top legal people in the industry, to start with the code instead of the pitch.

Regarding the perceived issue. At this time I am personally not working on this because in my opinion the threat to the network by having this mining pattern happening is smaller than the threat to freedom and permissionless participation that a hard fork of the protocol in the direction you have been suggesting would bring, and there is not enough data to make a decision. I have [encouraged](https://github.com/dogecoin/dogecoin/discussions/2591?sort=top#discussioncomment-1515921) a deeper look into the workings of [Digishield code](https://github.com/dogecoin/dogecoin/blob/31afd133119dd2e15862d46530cb99424cf564b0/src/dogecoin.cpp#L50-L66) because I suspect the continuous retargeting is what makes this possible at the 10%+ scale rather than have this at a much lower scale, but I have not seen any actual results or had indication that anyone has done any research at all.

**Reply by didlie** [reply]
A quick revision should be made and an update must be enforced. 2 or 3 blocks a minute is creating excessive dilution . Other steps must be taken to control dilution. And rebranding is a must. Half the world has cultural abhorrence  of dogs.

**Reply by patricklodder** [reply]
I still don't see any data, just your opinion. 

Re: dilution. I recently did the math on the blocks between Dec 6th 2020 to Dec 6th 2021 at midnight UTC and I came to the following:

- Block A: 3510596 
  - timestamp: `1607212809`
- Block B: 4008505
  - timestamp: `1638748693`
- Time delta: `1638748693 - 1607212809` = `31535884` seconds = 525598.06 minutes
- Block delta: `4008505 - 3510596` = `497909`
- Expected subsidy over 525598.06 minutes (rounded up) = `525599 * 10000` = 5,255,990,000 DOGE
- Realized subsidy over 497909 blocks = `497909 * 10000` = 4,979,090,000 DOGE
- Delta between realized and expected subsidy:  `4979090000 - 5255990000 ` = negative 276,900,000 DOGE

So, there is no excessive dilution, there is insufficient dilution. At this point, I'm begging you: please do some research before FUD!

**Reply by didlie** [reply]
It appears different in the explorer

**Reply by michilumin** [reply]
> And rebranding is a must. Half the world has cultural abhorrence of dogs.

I'd love to see the actual data on this one, not just an off the cuff anecdote. And the supporting counter that more than half the world will somehow get behind "duke coin".

otherwise, on the deep troll front, i give this a 7 out of 10. Not bad. I'd have made the branding thing less sweeping or saved your payload a bit longer, but not bad.

**Comment by didlie** [comment]
Per the discussion regarding data spamming by miners, see the attached transaction example.
![IMG_20211209_212334](https://user-images.githubusercontent.com/31107452/145507088-3c912bee-0bac-4ef0-9339-2d863cb63a32.jpg)

**Reply by patricklodder** [reply]
looks like 6 inputs spent into 2 output. This is good for memory. Not spam. It reduces the unspent transaction output db.

**Comment by didlie** [comment]
The best simple mathmatical solution to the necessary dilution of Dogecoin may be a network match of fees paid per transaction. The flat 10,000/55 sec dilution is exploitative. --- The incentive to mine empty blocks will be removed, dilution will increase linearly as transactions increase (as it should), and holders will experience near-term appreciation, encouraging further adoption.
